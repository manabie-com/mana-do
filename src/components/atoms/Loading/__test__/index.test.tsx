import React from 'react';
import {render} from '@testing-library/react';
import {Loading} from "../index";

describe('<Loading /> render', () => {
    it('should match snapshot', function () {
        const Element = render(<Loading isLoading/>);
        expect(Element.container.firstChild).toMatchSnapshot();
    });
})