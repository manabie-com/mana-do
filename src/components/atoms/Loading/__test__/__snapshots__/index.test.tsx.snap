// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> render should match snapshot 1`] = `
<div
  class="lds-dual-ring"
/>
`;

exports[`<Loading /> render should match snapshot 1`] = `
<div
  class="lds-dual-ring"
/>
`;
